From 38cc3510ab84783de3018f8997d4da9e1e6516b7 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Mon, 13 Aug 2018 18:47:35 -0700
Subject: [PATCH 25/44] Revert "debuggerd: Resolve tombstoned missing O_CREAT
 mode"

I merged this without it being ready. My bad...
This reverts commit f331e7fc8e6a0573790d7b436df0542a53e1190d.
---
 debuggerd/tombstoned/tombstoned.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debuggerd/tombstoned/tombstoned.cpp b/debuggerd/tombstoned/tombstoned.cpp
index e5d62b1b8..15ae40624 100644
--- a/debuggerd/tombstoned/tombstoned.cpp
+++ b/debuggerd/tombstoned/tombstoned.cpp
@@ -112,7 +112,7 @@ class CrashQueue {
 
   std::pair<std::string, unique_fd> get_output() {
     std::string path;
-    unique_fd result(openat(dir_fd_, ".", O_CREAT | O_WRONLY | O_APPEND | O_TMPFILE | O_CLOEXEC, 0640));
+    unique_fd result(openat(dir_fd_, ".", O_WRONLY | O_APPEND | O_TMPFILE | O_CLOEXEC, 0640));
     if (result == -1) {
       // We might not have O_TMPFILE. Try creating with an arbitrary filename instead.
       static size_t counter = 0;
-- 
2.17.0

